File: src/test/java/day5/IfYouGiveASeedAFertilizerTest.java
package day5;

import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertEquals;

class IfYouGiveASeedAFertilizerTest {

    @Test
    void firstStar() {
        IfYouGiveASeedAFertilizer ifYouGiveASeedAFertilizer = new IfYouGiveASeedAFertilizer();
        assertEquals(35, ifYouGiveASeedAFertilizer.firstStar());
    }

    @Test
    void secondStar() {
        IfYouGiveASeedAFertilizer ifYouGiveASeedAFertilizer = new IfYouGiveASeedAFertilizer();
        assertEquals(46, ifYouGiveASeedAFertilizer.secondStar());
    }
}

File: src/test/kotlin/day05/IfYouGiveASeedAFertilizerTest.kt
package day05

import org.junit.jupiter.api.Assertions.assertEquals
import org.junit.jupiter.api.Test

internal class IfYouGiveASeedAFertilizerTest {
    @Test
    fun firstStar() {
        assertEquals(35, IfYouGiveASeedAFertilizer().firstStar())
    }

    @Test
    fun secondStar() {
        assertEquals(46, IfYouGiveASeedAFertilizer().secondStar())
    }
}